chromium_src/third_party/boringssl/src/crypto/curve25519/curve25519.c

```diff
@@ -0,0 +1,80 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
```

Comment:
The upstream file seems to be:

```
third_party/boringssl/src/crypto/curve25519/curve25519.cc
```

However this one is:
```
chromium_src/third_party/boringssl/src/crypto/curve25519/curve25519.c
```

Is there any particular reason to mark this as a C file, rather than do like upstream and use a C++ file?

Reply:
upstream is 
`third_party/boringssl/src/crypto/curve25519/curve25519.c`

Reply:
Thanks to @goodov for the link
https://source.chromium.org/chromium/_/boringssl/boringssl/+/5813c2c10c73d800f1b0d890a7d74ff973abbffc
This is not yet in master

Reply:
Ah, fair enough.